{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-08T03:18:00.000+0000",
      "latitude": 51.29443998873175,
      "estimatedRestoration": "2022-03-08T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP6498'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP6498",
      "reference": "JP6498",
      "name": "RG23 & RG26 Areas",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG23 8PF",
        "RG26 5HJ",
        "RG26 5RD",
        "RG26 5RH",
        "RG26 5TD"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.1538965773275407,
          51.29443998873175
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-03-08T02:00:00.000+0000",
      "UUID": "e48044dd-365c-4547-b1f2-6792f65777a3",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.1538965773275407,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-08T07:19:00.000+0000",
      "latitude": 51.5044157937329,
      "estimatedRestoration": "2022-03-08T10:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JP6543'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JP6543",
      "reference": "JP6543",
      "name": "SL1 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SL1 1RQ",
        "SL1 2AY",
        "SL1 2BB",
        "SL1 2BD",
        "SL1 2BQ",
        "SL1 2BW",
        "SL1 2BX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.5899539076649813,
          51.5044157937329
        ],
        "type": "Point"
      },
      "UUID": "92293492-f1b7-4ca0-befd-2481c017876f",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.5899539076649813,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-07T15:33:00.000+0000",
      "latitude": 60.43653994960068,
      "estimatedRestoration": "2022-03-08T15:34:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '206530'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/206530",
      "reference": "206530",
      "name": "ZE2 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "ZE2 9RD"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.3583542717960795,
          60.43653994960068
        ],
        "type": "Point"
      },
      "UUID": "f9e6e92a-355e-41a7-839a-c4db790fe2c9",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3583542717960795,
      "resolved": false
    }
  ]
}
